compiler/src/dotty/tools/dotc/reporting/ErrorMessageID.scala

```diff
@@ -1,187 +1,190 @@
+  case ImplicitCaseClassID // errorNumber: 11 
+  case ImplicitClassPrimaryConstructorArityID // errorNumber: 12
+  case ObjectMayNotHaveSelfTypeID // errorNumber: 13
+  case TupleTooLongID extends ErrorMessageID(isActive = false) // errorNumber: 14
+  case RepeatedModifierID // errorNumber: 15
+  case InterpolatedStringErrorID // errorNumber: 16
+  case UnboundPlaceholderParameterID // errorNumber: 17
+  case IllegalStartSimpleExprID // errorNumber: 18
+  case MissingReturnTypeID // errorNumber: 19
+  case YieldOrDoExpectedInForComprehensionID // errorNumber: 20
+  case ProperDefinitionNotFoundID // errorNumber: 21
+  case ByNameParameterNotSupportedID // errorNumber: 22
+  case WrongNumberOfTypeArgsID // errorNumber: 23
+  case IllegalVariableInPatternAlternativeID // errorNumber: 24
+  case IdentifierExpectedID // errorNumber: 25
+  case AuxConstructorNeedsNonImplicitParameterID // errorNumber: 26
+  case VarArgsParamMustComeLastID // errorNumber: 27
+  case IllegalLiteralID // errorNumber: 28
+  case PatternMatchExhaustivityID // errorNumber: 29
+  case MatchCaseUnreachableID // errorNumber: 30
+  case SeqWildcardPatternPosID // errorNumber: 31
+  case IllegalStartOfSimplePatternID // errorNumber: 32
+  case PkgDuplicateSymbolID // errorNumber: 33
+  case ExistentialTypesNoLongerSupportedID // errorNumber: 34
+  case UnboundWildcardTypeID // errorNumber: 35
+  case DanglingThisInPathID extends ErrorMessageID(isActive = false) // errorNumber: 36
+  case OverridesNothingID // errorNumber: 37
+  case OverridesNothingButNameExistsID // errorNumber: 38
+  case ForwardReferenceExtendsOverDefinitionID // errorNumber: 39
+  case ExpectedTokenButFoundID // errorNumber: 40
+  case MixedLeftAndRightAssociativeOpsID // errorNumber: 41
+  case CantInstantiateAbstractClassOrTraitID // errorNumber: 42
+  case UnreducibleApplicationID // errorNumber: 43
+  case OverloadedOrRecursiveMethodNeedsResultTypeID // errorNumber: 44
+  case RecursiveValueNeedsResultTypeID // errorNumber: 45
+  case CyclicReferenceInvolvingID // errorNumber: 46
+  case CyclicReferenceInvolvingImplicitID // errorNumber: 47
+  case SuperQualMustBeParentID // errorNumber: 48
+  case AmbiguousReferenceID // errorNumber: 49
+  case MethodDoesNotTakeParametersId // errorNumber: 50
+  case AmbiguousOverloadID // errorNumber: 51
+  case ReassignmentToValID // errorNumber: 52
+  case TypeDoesNotTakeParametersID // errorNumber: 53
+  case ParameterizedTypeLacksArgumentsID // errorNumber: 54
+  case VarValParametersMayNotBeCallByNameID // errorNumber: 55
+  case MissingTypeParameterForID // errorNumber: 56
+  case DoesNotConformToBoundID // errorNumber: 57
+  case DoesNotConformToSelfTypeID // errorNumber: 58
+  case DoesNotConformToSelfTypeCantBeInstantiatedID // errorNumber: 59
+  case AbstractMemberMayNotHaveModifierID // errorNumber: 60
+  case TopLevelCantBeImplicitID extends ErrorMessageID(isActive = false) // errorNumber: 61
```

Comment:
Related to the other top level implicit comment this is also still referenced in one place, but I think it's dead code as I'm unable to reproduce it or get anything to touch that branch in

https://github.com/lampepfl/dotty/blob/53f5531e7e31eca02b3db158c8c384e713b4542a/compiler/src/dotty/tools/dotc/typer/Checking.scala#L477

Reply:
FYI there's no longer reference to this Error https://github.com/lampepfl/dotty/pull/14935